src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -39,6 +49,10 @@
                                     <div class="tooltip-inner ez-tooltip__inner"></div>
                                </div>`,
                 });
+
+                $(tooltipNode).on('inserted.bs.tooltip', (event) => {
```

Comment:
If this is after the tooltip is set it will be undefined on first time the tooltip is added?